playbook/app/pb_kits/playbook/pb_typeahead/components/MultiValue.jsx

```diff
@@ -28,19 +29,30 @@ const MultiValue = (props: Props) => {
         className="text_input_multivalue_container"
         {...props}
     >
-      <If condition={imageUrl}>
-        <FormPill
-            avatarUrl={imageUrl}
+      <If condition={multiKit === 'badge'}>
+        <Badge
             closeProps={removeProps}
-            marginRight="xs"
-            name={label}
-        />
-        <Else />
-        <FormPill
-            closeProps={removeProps}
-            marginRight="xs"
+            removeIcon
             text={label}
+            variant="primary"
         />
+        <Else />
+        <If condition={imageUrl}>
+          <FormPill
+              avatarUrl={imageUrl}
+              closeProps={removeProps}
+              marginRight="xs"
+              name={label}
+              size={multiKit === 'smallPill' ? 'small' : ''}
```

Comment:
I think `size={multiKit === 'smallPill' && 'small'}` I think this will work for this. I could be wrong.